public/src/pages/StudentPage.tsx

```diff
@@ -86,6 +89,23 @@ export class StudentPage extends ViewPage {
         </div >;
     }
 
+    public async archive(navInfo: INavInfo<any>): View {
```

Comment:
Maybe rename to `hide`?

Reply:
Renamed to `courseList` because it is the course list it is responsible for.